### PR TITLE
fix: display same Avatar from WalletButton in dashboard page

### LINKED ISSUE #928: profile pic is never defined
Capture d’écran 2024-11-07 à 10 59 15

On our new dashboard page, the avatar is never defined, we want it to be the same image we have on the connection button and the `Avatar.tsx` component. 

You can use 

```tsx
  const { data: profileData } = useStarkProfile({ address });
```

### PR DESCRIPTION
**Pull request by wheval**
# Added alt text for the avatar image and displayed the avatar on the dashboard page.

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix

Resolves: #928 

## Other information

Here's how it looks.
Screenshot (44)

## Summary by CodeRabbit

- **New Features**
	- Enhanced accessibility for the `Avatar` component with an `alt` attribute for user profile images.
	- Updated `ProfileCard` component to simplify data flow by directly using the user's address for rendering profile pictures.

- **Bug Fixes**
	- Improved conditional rendering for profile pictures in the `ProfileCard`, ensuring default icons are displayed when necessary.

### CODE REVIEW

**Review by fricoben — APPROVED** [review]
lgtm

File: components/UI/profileCard/profileCard.tsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ const ProfileCard: FunctionComponent<ProfileCardProps> = ({
   const formattedAddress = (
     identity.owner.startsWith("0x") ? identity.owner : `0x${identity.owner}`
   ) as Address;
-  const { data: profileData } = useStarkProfile({ address: formattedAddress });
+  const { address, status: accountStatus } = useAccount();
```
Comment (fricoben) [inline components/UI/profileCard/profileCard.tsx]: You don't use `address` or `accountStatus` here, do you ?

File: components/UI/profileCard/profileCard.tsx [hunk 2]
```diff
@@ -112,8 +113,8 @@ const ProfileCard: FunctionComponent<ProfileCardProps> = ({
     <div className={styles.dashboard_profile_card}>
       <div className={styles.left}>
         <div className={styles.profile_picture_div}>
-          {profileData?.profilePicture ? (
-            <img src={profileData.profilePicture} className="rounded-full" />
+          { formattedAddress && formattedAddress?.length !== 0 ? ( // show the avatar of the address in the URL
```
Comment (fricoben) [inline components/UI/profileCard/profileCard.tsx]: simplify 

{formattedAddress?.length ? (
    <Avatar width="120" address={formattedAddress} />
...

**Review by fricoben — CHANGES_REQUESTED** [review]
Some small changes

File: components/UI/profileCard/profileCard.tsx (hunk 2 above)
Comment (wheval) [reply components/UI/profileCard/profileCard.tsx]: how did i not see that, on it..

**Comment by wheval** [comment]
@fricoben please review, thanks!

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**